src/BitmapText.cpp

```diff
@@ -770,12 +768,9 @@ void BitmapText::DrawPrimitives()
 		std::vector<RageVector3> vGlyphJitter;
 		if( m_bJitter )
 		{
-			int iSeed = std::lrint( RageTimer::GetTimeSinceStartFast()*8 );
-			RandomGen rnd( iSeed );
-
 			for( unsigned i=0; i<m_aVertices.size(); i+=4 )
 			{
-				RageVector3 jitter( rnd()%2, rnd()%3, 0 );
+				RageVector3 jitter( (GetRandomFloat()%2), (GetRandomFloat()%3), 0 );
```

Comment:
rnd() here is a RandomInt and not a random float? This would be a difference in behavior.